docs/faqs/NuGet-FAQ.md

```diff
@@ -232,15 +232,17 @@ If that fails, try the [tips mentioned in this StackOverflow post](http://stacko
 
 ## What is the API endpoint for nuget.org?
 
-To use  nuget.org as a package repository with NuGet clients, you would need to use the following V3 API endpoint: 
+To use nuget.org as a package repository with NuGet clients, you would need to use the following V3 API endpoint: 
```

Comment:
Thoughts on `you would need to use` -> `you should use` / `you need to use`?

Reply:
`you should use` is best I think.

Reply:
I like “you need” better

Reply:
You don't need to use. 3.x and new support the V2 protocol. People know this.